pkg/providers/v1/instances_v2.go

```diff
@@ -82,8 +83,11 @@ func (c *Cloud) getAdditionalLabels(ctx context.Context, zoneName string, instan
 	// If topology labels are already set, skip.
 	if _, ok := existingLabels[LabelNetworkNodePrefix+"1"]; !ok {
 		nodeTopology, err := c.instanceTopologyManager.GetNodeTopology(ctx, instanceType, region, instanceID)
+		// We've seen some edge cases where this functionality is problematic, so swallowing errors and logging
```

Comment:
What type of errors are possible?

Reply:
This is a super defensive check intended to check any unknown errors to avoid any possibility that it gets rolled back if we find new edge cases. What comes to mind is any other node variant that's not handled gracefully or race conditions where the instance exists, but maybe this API is eventually consistent?

Reply:
Other than that, there's nothing known here. Makes this best effort to avoid the unknown and breaking nodes getting initialized.